Onboarding (first part)

EnzoVentura opened the pull request.

Commit:
feat: next button wip & name page

Commit:
fix: onboarding layout

Commit:
feat: name introduction page

Commit:
feat: birthday page

Commit:
feat: gender page

AurelienV-42 commented on assets/logo/icon.png:
Ça c'est l'icône de l'app, il faut que ça soit en 1024x1024, si tu veux check pour la grosseur du trait je suis chaud mais on ne peut pas changer la résolution

AurelienV-42 commented on src/components/inputs/MyBirthdayInput.tsx:
```diff
@@ -0,0 +1,53 @@
+  );
+
+  return (
+    <TextInput
```
pourquoi tu ne prends pas MyInput? Tu me l'as dit mais je me souviens plus

AurelienV-42 replied:
@EnzoVentura

EnzoVentura replied:
J'utilise désormais MyInput

AurelienV-42 commented on src/locales/fr.json:
```diff
@@ -6,5 +6,8 @@
     "description5": "Le reste se passe dans la vie réelle."
+  },
+  "name": {
+    "title": "Comment vous appelez-vous ?"
```
Fais pas le français t'inquiète osef, on fera toute la trad avec une IA à la fin

AurelienV-42 commented on src/pages/onboarding/Birthday.tsx:
```diff
@@ -0,0 +1,40 @@
+  return (
+    <MyOnboardingLayout
+      onNextPress={() => {
+        navigation.navigate("Gender");
```
Du coup le but est de pouvoir récupérer le cont OnboardingViews de OnboardingNavigator.tsx pour aller à l'écran d'après. Comme ça juste en changeant l'array on swipe les écrans

AurelienV-42 commented on src/pages/onboarding/Birthday.tsx:
```diff
@@ -0,0 +1,40 @@
+          <MyText className="text-3xl font-semibold">
+            {t("birthday.title")}
+          </MyText>
+          <MyText className="text-gray-500 text-xs">
```
les gris faudra les revoir parce que ruben en a mis plein dans le figma

AurelienV-42 commented on src/pages/onboarding/Gender.tsx:
```diff
@@ -0,0 +1,83 @@
+      className={`w-full flex-row items-center p-4 rounded-3xl ${
+        isSelected ? "bg-gradient-primary-0" : "bg-gray-400"
+      }`}
+      style={{ gap: 12 }}
```
normal qu'il y ait un gap là ?

EnzoVentura replied:
Il me semble que c'était pour du debug, il n'y est plus

AurelienV-42 commented on src/pages/onboarding/Gender.tsx:
```diff
@@ -0,0 +1,83 @@
+          size: 24,
+          color: "white",
+          weight: "bold",
+        })}
```
Euh ça c'est pas good, tu peux rename icon par IconComponent (avec une majuscule pour capter c'est un Component) et faire {Icon && <Icon />}

AurelienV-42 committed:
fix: improve icons and wip gradient

AurelienV-42 committed:
chore: better logo cropped
shadow purple
bump anim on next button
back button in the header
improve the MyButton but WIP

AurelienV-42 committed:
refacto: haptics and MyGradient

AurelienV-42 committed:
feat: create account / login

AurelienV-42 committed:
fix: account phone

Commit:
feat: next verfications

Commit:
feat: phone & share pages

AurelienV-42 reviewed (changes requested):
test

Commit:
fix: use MyInput in MyBirthdayInput

Commit:
feat: phone input
